var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -345,6 +345,9 @@ def nvariant_bool(feature):
             '-DBUILD_TESTING:BOOL=OFF',
             '-DOpenGL_GL_PREFERENCE:STRING=LEGACY']
 
+        if spec.satisfies('@master'):
```

Comment:
I would also include versions greater than 5.10.1

Reply:
That should be good.  i.e. `if '@5.11:,master' in spec:`

Reply:
@chuckatkins Why do we need to specify master? spack could do some git queries to see where master is located. That would be a nice improvement.

Reply:
I'm surprised too: I thought spack consider master > any release number

Reply:
> I would also include versions greater than 5.10.1

I do not think that a patch release will change the VTK versions.

I found the commit introducing this at https://gitlab.kitware.com/vtk/vtk/-/commit/930d460e37c3993814b94298b956d2f2e0b47bc2

This commit was introduced to master here https://gitlab.kitware.com/paraview/paraview/-/merge_requests/5522, the master commit is https://gitlab.kitware.com/paraview/paraview/-/commit/0e42156a2b9b4d6066bb1510231073de723cf6b1.

I wonder if there is any way in Spack to say: if "after this particular commit"

if we do not have such tool perhaps we can consider enabling this for everything after 5.10.1 (inclusive, even 5.10.1 that does not need it) considering that this is harmless for 5.10.1.

Reply:
What about spec.satisfies('@5.10.2:'). That would include master according to https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison

Reply:
Yes, 5.11 works to do the same thing so there is no need to other versions. I was trying to come up with an earlier version than the current master but later than the released version.